### PR TITLE
[Enhance] Support load annotations from ceph

### PR DESCRIPTION
**Pull request by ZCMax**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Support load annotations from ceph for datasets and dbsampler

## Modification
Support load annotations from ceph for datasets and dbsampler

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by Tai-Wang — COMMENTED** [review]
Please also add related documentation.

**Comment by ZCMax** [comment]
> Please also add related documentation.

done

File: docs/en/tutorials/backends_support.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Tutorial 7: Backends Support
+We support different file client backends: Disk, Ceph and IMDB, etc. Here is an example of Ceph loading and saving modification in config.
```
Comment (Tai-Wang) [inline docs/en/tutorials/backends_support.md:2]: Here is an example of how to modify configs for Ceph-based data loading and saving.

File: docs/en/tutorials/backends_support.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+## Load data and annotations from Ceph
+
+We support load data and generated annotation info files (pkl and json) from Ceph:
```
Comment (Tai-Wang) [inline docs/en/tutorials/backends_support.md:6]: loading

File: docs/en/tutorials/backends_support.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        type='NoStemRegNet',
+        arch='regnetx_1.6gf',
+        init_cfg=dict(
+            type='Pretrained', checkpoint='s3://openmmlab/checkpoints/mmdetection3d/regnetx_1.6gf'), # replace the path to your pretrained model path in Ceph
+        ...
+```
+
+## Load checkpoint from Ceph
+```python
+# replace the path to your checkpoint path in Ceph
```
Comment (Tai-Wang) [inline docs/en/tutorials/backends_support.md:112]: replace the path with your checkpoint path on Ceph

File: docs/en/tutorials/backends_support.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+## Save the training log into Ceph
+The training log will be backed up to the specified Ceph path after training.
+
+```python
+log_config = dict(
+    interval=50,
+    hooks=[
+        dict(type='TextLoggerHook', out_dir='s3://openmmlab/mmdetection3d'),
+        # dict(type='TensorboardLoggerHook')
```
Comment (Tai-Wang) [inline docs/en/tutorials/backends_support.md:141]: If this setting is not used, we can remove it. Please also check other places (like the below one).

File: docs/zh_cn/tutorials/backends_support.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
+# Tutorial 7: 后端支持
+我们支持不同的文件客户端后端：磁盘、Ceph 和 IMDB 等。下面是 Ceph 在配置中加载和保存修改的示例。
```
Comment (Tai-Wang) [inline docs/zh_cn/tutorials/backends_support.md:2]: 下面是修改配置使之从 Ceph 加载和保存数据的示例。

File: docs/zh_cn/tutorials/backends_support.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    ])# yapf:enable
+```
+您还可以通过设置 `keep_local = False` 备份到指定的 Ceph 路径后删除本地训练日志。
+
+```python
+log_config = dict(
+    interval=50,
+    hooks=[
+        dict(type='TextLoggerHook', out_dir='s3://openmmlab/mmdetection3d'', keep_local=False),
+        # dict(type='TensorboardLoggerHook')
```
Comment (Tai-Wang) [inline docs/zh_cn/tutorials/backends_support.md:151]: Also fix the Chinese version as the comments for the English version.

File: docs/en/tutorials/backends_support.md (hunk 1 above)
Comment (ZwwWayne) [inline docs/en/tutorials/backends_support.md:2]: Empty line above.

Comment (ZwwWayne) [inline docs/en/tutorials/backends_support.md:2]: IMDB or LMDB?

File: mmdet3d/datasets/lyft_dataset.py [hunk 7]
```diff
@@ -116,7 +118,7 @@ def load_annotations(self, ann_file):
         Returns:
             list[dict]: List of annotations sorted by timestamps.
         """
-        data = mmcv.load(ann_file)
+        data = mmcv.load(ann_file, file_format='pkl')
```
Comment (ZwwWayne) [inline mmdet3d/datasets/lyft_dataset.py:121]: May leave comments to explain the necessity.

**Review by ZwwWayne — APPROVED** [review]

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**